.github/workflows/on-push.yml

```diff
@@ -78,6 +78,7 @@ jobs:
         with:
           path: ${{ env.CACHE_NODE_MODULES_PATH }}
           key: build-${{ hashFiles('**/yarn-lock.json') }}
+      - run: ls node_modules/@antora
```

Comment:
Is that needed?

Reply:
that's from the tmp commit that I'm adding, gonna remove. For some reason build fails in CI environment, which is not reproducible neither locally, nor in isolated containers